nautobot_circuit_maintenance/handle_notifications/sources.py

```diff
@@ -38,7 +39,8 @@
 class MaintenanceNotification(BaseModel):
     """Representation of all the data related to a Maintenance Notification."""
 
-    source: str
+    msg_id: bytes
+    source: "Source"
```

Comment:
here we lose track of the Source name, and add hardcode this string `Source`. Maybe we were not actually using the `source` before, but for which purpose we use this string now?

Reply:
follow-up question. If we decided to move `Source` before this part of the code, could we have used `source: Source` directly, no? and doing so, could we have omitted the `update_forward_refs()`?

Reply:
Yes, that would be an alternate approach, assuming there aren't forward references to `MaintenanceNotification` within the `Source` class that would be equally problematic.

Reply:
Turned out to be easy enough to move MaintenanceNotification below Source, so I've done so.